src/Webapi.re

```diff
@@ -12,3 +12,5 @@ module File = {
   include FileRe;
   module Url = UrlRe;
 };
+
+module Url = UrlRe;
```

Comment:
It's just preference. =)
It `URL` doesn't seem to be part of `File`. But at the same time I didn't want to just move it in case it breaks stuff.

Reply:
I think it was put there because the parts that were implemented are defined in the File specification. But if this is added the old alias should be deprecated I think. I can do that myself though.